updater/lib/dependabot/sorbet/runtime.rb

```diff
@@ -0,0 +1,29 @@
+require "sorbet-runtime"
+
+module Dependabot
+  module Sorbet
+    module Runtime
+      class InformationalError < StandardError; end
+
+      def self.silently_report_errors!
+        T::Configuration.call_validation_error_handler = lambda do |sig, opts|
+          error = InformationalError.new(build_message(sig, opts))
```

Comment:
This is the default handler: https://github.com/sorbet/sorbet/blob/799c78d6d04231f10062c773a58f4bbc8c6bb7b3/gems/sorbet-runtime/lib/types/configuration.rb#L295-L297

Looks like we can just extract `:pretty_message` and use that instead of building our own?